packages/rum-core/src/domain/privacy.ts

```diff
@@ -150,6 +153,43 @@ export function shouldMaskNode(node: Node, privacyLevel: NodePrivacyLevel) {
   }
 }
 
+export function shouldMaskAttribute(
+  tagName: string,
+  attributeName: string,
+  nodePrivacyLevel: NodePrivacyLevel,
+  configuration: RumConfiguration,
+  attributeValue?: string | null
+) {
+  if (nodePrivacyLevel === NodePrivacyLevel.MASK || nodePrivacyLevel === NodePrivacyLevel.MASK_UNLESS_ALLOWLISTED) {
+    if (
+      attributeName !== PRIVACY_ATTR_NAME &&
+      !STABLE_ATTRIBUTES.includes(attributeName) &&
+      attributeName !== configuration.actionNameAttribute
+    ) {
```

Comment:
Same style nit here; please invert this condition and return early:

```suggestion
    if (
      attributeName === PRIVACY_ATTR_NAME ||
      STABLE_ATTRIBUTES.includes(attributeName) ||
      attributeName === configuration.actionNameAttribute
    ) {
      return false
    }
```